doc/source/readers.rst

```diff
@@ -126,20 +128,26 @@ following attributes are standardized across all readers:
-      geodetic coordinates (i.e. altitude is normal to the surface).
-    * ``satellite_nominal_longitude/latitude/altitude``: Centre of the station keeping box (a confined area in which
-      the satellite is actively maintained in using maneuvres). Inbetween major maneuvres, when the satellite
-      is permanently moved, the nominal position is constant.
-    * ``nadir_longitude/latitude``: Intersection of the instrument's Nadir with the surface of the earth. May differ
-      from the actual satellite position, if the instrument is poiting slightly off the axis (satellite, earth-centre).
-      If available, this should be used to compute viewing angles etc. Otherwise, use the actual satellite position.
-    * ``projection_longitude/latitude/altitude``: Projection centre of the re-projected data. This should be used to
-      compute lat/lon coordinates. Note that the projection centre can differ considerably from the actual satellite
-      position. For example MSG-1 was at times positioned at 3.4 degrees west, while the image data was re-projected
-      to 0 degrees.
-    * [DEPRECATED] ``satellite_longitude/latitude/altitude``: Current position of the satellite at the time of observation
-      in geodetic coordinates.
+    * ``satellite_actual_longitude/latitude/altitude``: Current position of the satellite at the
+      time of observation in geodetic coordinates (i.e. altitude is relative and normal to the
+      surface of the ellipsoid).
+    * ``satellite_nominal_longitude/latitude/altitude``: Centre of the station keeping box (a
+      confined area in which the satellite is actively maintained in using maneuvres). Inbetween
+      major maneuvres, when the satellite is permanently moved, the nominal position is constant.
+    * ``nadir_longitude/latitude``: Intersection of the instrument's Nadir with the surface of the
+      earth. May differ from the actual satellite position, if the instrument is poiting slightly
+      off the axis (satellite, earth-centre). If available, this should be used to compute viewing
+      angles etc. Otherwise, use the actual satellite position.
+    * ``projection_longitude/latitude/altitude``: Projection centre of the re-projected data. This
+      should be used to compute lat/lon coordinates. Note that the projection centre can differ
+      considerably from the actual satellite position. For example MSG-1 was at times positioned
+      at 3.4 degrees west, while the image data was re-projected to 0 degrees.
+    * [DEPRECATED] ``satellite_longitude/latitude/altitude``: Current position of the satellite at
+      the time of observation in geodetic coordinates.
+
+    .. note:: Longitudes and latitudes are given in degrees, altitude in meters. For use in
+              pyorbital, the altitude has to be converted to kilometers, see for example
+              `get_observer_look`_.
```

Comment:
@djhoese you did get references across packages to work, right ?

Reply:
Ah yes, as long as pyorbital's documentation has been regenerated recently. I'm not sure if it needs to include the `intersphinx` extension in its `conf.py` either. We'd also have to add `pyorbital` to the `intersphinx_mapping` dictionary at the end of Satpy's `conf.py`. Then this could be changed to:

```
:func:`pyorbital:get_observer_look`
```

I think that should work. If not then changing the function name to `pyorbital.get_observer_look` (or the full path to the function) might work (`:` versus `.`).

Reply:
I just rebuilt the pyorbital doc now.

Reply:
Looks good on the pyorbital front. The main thing you look for is if it has an `objects.inv` file which pyorbital does: https://pyorbital.readthedocs.io/en/latest/objects.inv

You can then analyze it on the command line by doing:

```
python -m sphinx.ext.intersphinx https://pyorbital.readthedocs.io/en/latest/objects.inv
```

It'll give you a list of all the references you can make (documents, functions, classes, etc).

Reply:
Nice, thanks for the detailed instructions. Works fine!